app/models/container.rb

```diff
@@ -51,4 +49,13 @@ def disconnect_inv
     self.ems_id = nil
     save
   end
+
+  # Needed for metrics
+  def my_zone
```

Comment:
@zeari this seems duplicated code (on all the other entities) that may deserve a mixin or something.